server/ig-main.js

```diff
@@ -0,0 +1,63 @@
+const Client = require('instagram-private-api').V1;
+const device = new Client.Device('eatifyjohn');
+const path = require('path');
+const cookiePath = path.join(__dirname, '/cookies/eatifyjohn.json');
+const storage = new Client.CookieFileStorage(cookiePath);
+
+function IG() {
+
+}
+
+IG.prototype.getAccountByName = function(username, session) {
```

Comment:
I'm leaning towards this since I'll have very similar functions for both IG and postgres and didn't want to get confused. I was also going to add some variables to the IG and postgres objects so that I could keep track of things while doing asynchronous operations, much like i did with my twitter object.